src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/Todo.razor.cs

```diff
@@ -9,29 +9,24 @@ public partial class Todo
 
     public bool IsBussy { get; set; } = false;
```

Comment:
This variable name has an incorrect spelling.
btw, it's better to name such a variable to `IsLoading`